File: android-java/app/src/main/java/cn/shengwang/convoai/quickstart/KeyCenter.java
package cn.shengwang.convoai.quickstart;

/**
 * KeyCenter
 * Load values from BuildConfig, which are populated from env.properties at build time
 */
public class KeyCenter {
    public static final String APP_ID = BuildConfig.APP_ID;
    public static final String APP_CERTIFICATE = BuildConfig.APP_CERTIFICATE;
    public static final String LLM_API_KEY = BuildConfig.LLM_API_KEY;
    public static final String LLM_URL = BuildConfig.LLM_URL;
    public static final String LLM_MODEL = BuildConfig.LLM_MODEL;
    public static final String TTS_BYTEDANCE_APP_ID = BuildConfig.TTS_BYTEDANCE_APP_ID;
    public static final String TTS_BYTEDANCE_TOKEN = BuildConfig.TTS_BYTEDANCE_TOKEN;
}

File: android-kotlin/app/src/main/java/cn/shengwang/convoai/quickstart/KeyCenter.kt
package cn.shengwang.convoai.quickstart

import cn.shengwang.convoai.quickstart.BuildConfig

object KeyCenter {
    // Shengwang App Credentials
    val APP_ID: String = BuildConfig.APP_ID
    val APP_CERTIFICATE: String = BuildConfig.APP_CERTIFICATE

    // LLM Configuration
    val LLM_API_KEY: String = BuildConfig.LLM_API_KEY
    val LLM_URL: String = BuildConfig.LLM_URL
    val LLM_MODEL: String = BuildConfig.LLM_MODEL

    // STT Configuration
    val STT_MICROSOFT_KEY: String = BuildConfig.STT_MICROSOFT_KEY
    val STT_MICROSOFT_REGION: String = BuildConfig.STT_MICROSOFT_REGION

    // TTS Configuration
    val TTS_MINIMAX_KEY: String = BuildConfig.TTS_MINIMAX_KEY
    val TTS_MINIMAX_MODEL: String = BuildConfig.TTS_MINIMAX_MODEL
    val TTS_MINIMAX_VOICE_ID: String = BuildConfig.TTS_MINIMAX_VOICE_ID
    val TTS_MINIMAX_GROUP_ID: String = BuildConfig.TTS_MINIMAX_GROUP_ID
}
